docs/Marketplace.md

```diff
@@ -16,7 +18,24 @@ In this tutorial, you will:
 3. [Run Codex](#3-run-codex);
 4. [Buy and Sell Storage in the Marketplace](#4-buy-and-sell-storage-on-the-marketplace).
 
-We strongly suggest you to create a folder (e.g. `marketplace-tutorial`), and switch into it before beginning.
+To get things running smoothly, we recommend the following directory structure:
+
+```
+|
+|-- nim-codex (this is the current repo)
+|-- codex-contracts-eth
+└-- marketplace-tutorial
+```
+
+We will clone the `codex-contracts-eth` repository at a later stage, for now just create the `marketplace-tutorial` directory:
```

Comment:
this is done automatically and pulled in as a git submodule, do we really need to clone it separately?

Reply:
Ugh, I made that choice deliberately but honestly can't remember why anymore. :-/

Reply:
> this is done automatically and pulled in as a git submodule, do we really need to clone it separately?

right, you mean in the `nim-codex` via `vendors`. We could indeed take advantage of it, but on the other hand, having it separate may be easier to the newcomers - it somehow gives you clear visibility what is involved.

But saying that, later I will give it a try and check how does it feel. Maybe it will turn out to be better experience. The user may benefit from learning the structure of the nim-codex repo a bit better....

Reply:
I think it's one step less to go through, you just have to change to a different directory `cd vendor/codex-contracts-eth` :)